support-frontend/assets/helpers/internationalisation/countryGroup.js

```diff
@@ -103,20 +103,21 @@ const countryGroups: CountryGroups = {
 
 // ----- Functions ----- //
 
-function fromPath(path: string = window.location.pathname): ?CountryGroupId {
-  if (path === '/uk' || path.startsWith('/uk/')) {
+function fromUrl(): ?CountryGroupId {
```

Comment:
Not a major suggestion but we could make it more reusable by turning the the url a default parameter

```suggestion:
function fromUrl(url: string = window.location.href): ?CountryGroupId {
```

Reply:
Yep that's how it was before, but the only place it's use is in `detect` and all the other functions called in `detect` accept no arguments and depend just on browser state. So for me this one accepting a parameter is confusing. I can't think of a use case where you would want to pass in a URL other than the page the browser is on to figure out the geolocation